src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -171,6 +220,14 @@ function DetailsForm({
         type: 'VerticalLayout',
     };
 
+    const updateDetails = (details: Details) => {
```

Comment:
The reset endpoint config logic formally executed due to the dispatching of the `setDetails` action was migrated to this details form change handler. The reason for this highlighted in the comment above.